lib/winston-rollbar.js

```diff
@@ -70,9 +70,12 @@ Rollbar.prototype.log = function (level, msg, meta, callback) {
         }
     }
 
-    if (['warn','error'].indexOf(level) > -1 && (msg instanceof Error || meta instanceof Error)) {
+    if (['warn','error','critical'].indexOf(level) > -1 && (msg instanceof Error || meta instanceof Error)) {
```

Comment:
winston doesn't have a critical level by default.  If using syslog levels, it is 'crit', but this module doesn't support syslog levels anyway... yet :)